.changeset/warm-roses-lay.md

```diff
@@ -0,0 +1,6 @@
+---
+"@wso2is/admin.core.v1": minor
+"@wso2is/console": minor
```

Comment:
We do not add a new feature to console, don't we?

Reply:
We have removed the feature from extensions and added it as a separate feature in Console.

That said, any improvements, refactoring should go in a `minor` release.

Only bug fixes can go in `patch`.